.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.5
+org.eclipse.jdt.core.compiler.compliance=1.5
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
+org.eclipse.jdt.core.compiler.source=1.5
```

Comment:
Currently the project is set up to run on `J2SE-1.5` JRE version. Please, from inside Eclipse, change the used JRE to `JavaSE-1.8` (Right click on project -> Properties -> Java Build Path -> Libraries tab -> double click on JRE System Library). The `org.eclipse.jdt.core.prefs` file will be automatically updated.